sentry/src/main/java/io/sentry/NoOpHub.java

```diff
@@ -106,4 +106,14 @@ public void flush(long timeoutMillis) {}
   public IHub clone() {
     return instance;
   }
+
+  @Override
+  public SentryId captureTransaction(Transaction transaction, Object hint) {
+    return SentryId.EMPTY_ID;
+  }
+
+  @Override
+  public Transaction startTransaction(String name, TransactionContexts transactionContexts) {
+    return null;
```

Comment:
Without null check at compile time I'm afraid this could throw.
Nit: `NoOpTransaction` maybe? Not ideal, I agree.

Reply:
👍